concrete/config/concrete.php

```diff
@@ -547,7 +547,16 @@
              *
              * @var boolean
              */
-            'preview_image_popover' => true
+            'preview_image_popover' => true,
+            // SVG sanitization
+            'svg_sanitization' => [
+                // Enable the SVG sanitification?
+                'enabled' => true,
+                // Space-separated list of tags to be kept
+                'allowed-tags' => '',
+                // Space-separated list of attributes to be kept
+                'allowed-attributes' => '',
```

Comment:
@mlocati since all properties are with underscores, is there a reason these contain a dash?